scripts/vcf_site_level_filtering_wdl/JointVcfFiltering.wdl

```diff
@@ -137,6 +145,15 @@ workflow JointVcfFiltering {
 		Array[File] indels_variant_scored_vcf_index = ScoreVariantAnnotationsINDELs.output_vcf_index
 		Array[File] snps_variant_scored_vcf = ScoreVariantAnnotationsSNPs.output_vcf
 		Array[File] snps_variant_scored_vcf_index = ScoreVariantAnnotationsSNPs.output_vcf_index
+		Array[File?] monitoring_logs = flatten(
```

Comment:
Pondering if the pattern should be that any given workflow also call and output uber_monitor's summary file?

Reply:
I don't think I understand what this is asking

Reply:
Should I have JointVcfFiltering itself call summarize_task_monitor_logs and generate its own report in addition to providing the outputs for the parent workflow to summarize the logs.